README.md

```diff
@@ -444,6 +445,51 @@ Medium.
 - [Idea discussion](https://rust-lang.zulipchat.com/#narrow/channel/421156-gsoc/topic/Idea.3A.20Extend.20testing.20of.20.60std.3A.3Aarch.60.20intrinsics)
 - [t-libs/stdarch](https://rust-lang.zulipchat.com/#narrow/channel/208962-t-libs.2Fstdarch)
 
+### Add safety contracts
+
+**Description**
+
+We have taken a goal to
+[instrument the Rust standard library with safety contracts](https://rust-lang.github.io/rust-project-goals/2025h1/std-contracts.html).
+With this approach we are moving from informal comments specifying safety
+requirements on `unsafe` functions to executable Rust code.
+To prioritize which functions to equip with contracts first, we run a
+verification contest in the
+[verify-rust-std](https://github.com/model-checking/verify-rust-std) fork.
+In this contest,
+[challenges](https://model-checking.github.io/verify-rust-std/challenges.html)
+have been put up that request specific sets of functions to be equipped with
+contracts.
+
+Among others, we are currently looking for contributions towards the following challenges:
+- [Verify the memory safety of core intrinsics using raw pointers](https://model-checking.github.io/verify-rust-std/challenges/0002-intrinsics-memory.html)
+- [Memory safety of BTreeMap's `btree::node` module](https://model-checking.github.io/verify-rust-std/challenges/0004-btree-node.html)
+- [Safety of Methods for Atomic Types & Atomic Intrinsics](https://model-checking.github.io/verify-rust-std/challenges/0007-atomic-types.html)
+- [Contracts for SmallSort](https://model-checking.github.io/verify-rust-std/challenges/0008-smallsort.html)
+- [Safety of `NonZero`](https://model-checking.github.io/verify-rust-std/challenges/0012-nonzero.html)
+
+**Expected result**
+
+Safety contracts and harnesses as pull requests or fully merged.
```

Comment:
A set of safety contracts and harnesses have been implemented.

Reply:
Updated as suggested.